index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Mobile Portfolio</title>
+  <div class="header-items">
+    
+    <img src="img/My Logo.svg" alt="humburger" class="Logo" > 
+    <img src="img/Union.svg" alt="humburger"  class="Union" >
+  </div>
+
+</head>
+<body>
+  <header>
+    <div class="container">
+      <h2>I'm Edgah.<br>
+        Glad to see you! </h2>
+      <p> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+    <img src="img/LET’S CONNECT.svg" alt="humburger"><br><br>
+    <img src="img/social media icons.svg" alt="humburger">
```

Comment:
- Kindly use a ul tag to group the social icons and a li tag for each icon
- Please ensure each of the icons has a link to help users to connect to your socials and it is the correct way semantically